src/appleseed/renderer/kernel/rendering/final/uniformpixelrenderer.cpp

```diff
@@ -275,4 +276,31 @@ IPixelRenderer* UniformPixelRendererFactory::create(
     return new UniformPixelRenderer(m_factory, m_params, thread_index);
 }
 
+Dictionary UniformPixelRendererFactory::get_params_metadata()
+{
+    Dictionary metadata;
+    metadata.dictionaries().insert(
+        "samples",
+        Dictionary()
+            .insert("type", "int")
+            .insert("default", "64")
+            .insert("help", "Number of anti-aliasing samples"));
+
+    metadata.dictionaries().insert(
+        "force_antialiasing",
+        Dictionary()
+            .insert("type", "bool")
+            .insert("default", "false")
+            .insert("help", "TODO: Not documented yet"));
```

Comment:
When using 1 sample/pixel and this is disabled, samples are placed in the middle of the pixels.